srcs/python/kungfu/tensorflow/ops/collective.py

```diff
@@ -24,6 +24,11 @@ def all_reduce(t, op='sum'):
     return _op_lib.kungfu_all_reduce(t, op=op)
 
 
+def monitored_all_reduce(t, tree, op='sum'):
+    """Create a new all_reduce operator for given tensor and topology."""
+    return _op_lib.kungfu_monitored_all_reduce(t, tree, op=op)
```

Comment:
Should we provide a further description regarding the structure and usage of the `tree` parameter?